kratos/processes/process.h

```diff
@@ -124,6 +124,13 @@ class Process : public Flags
     {
     }
 
+    /// this function is designed for being called after ExecuteInitialize ONCE to
+    /// verify that the input is correct.
+    virtual int Check()
```

Comment:
why not `virtual bool Check()` ?

Reply:
other classes return ints
e.g. ModelPart, SolvingStrategy